### ISSUE TITLE
fix: Investigate possibility of scheduled builds disablement after 60 days of last repo commit

### ISSUE THREAD

**Issue by fiftydinar** [blue-build/github-action#68]
According to this [keepalive-workflow](https://github.com/gautamkrishnar/keepalive-workflow), users can get this message if there is no activity to the repo for 60+ days:

![105174930-4303e100-5b49-11eb-90ed-95a55697582f](https://github.com/user-attachments/assets/0c3dbc80-0837-4626-bc2e-3a77289a344c)

Which means that their image wouldn't get any updates if they didn't commit anything for 60+ days.
This is not a good thing. Users should be able to just set & forget their image.

However, analyzing the [Atomic Studio project](https://github.com/atomic-studio-org/Atomic-Studio), I can see that the last commit was done in 10th September at the time of writing, while the scheduled weekly build normally ran in 24th November, which is 75 days without any new commit.

So I'm not sure when this actually happens, on which scenarios & similar.

So we need to investigate if this needs to be solved at all:
- if there's no need to solve this issue, we can conclude & close this issue
- Otherwise, we should investigate the usage of [keepalive-workflow](https://github.com/gautamkrishnar/keepalive-workflow) in this action

**Comment by xynydev** [comment]
This workflow creates a dummy commit every 60 days, which might litter a users commit history. I'm not really sure it's possible to integrate into the action either, so the path to implementing this might be changes in the template. 

Anyways, I always get an email when one of my workflows is almost disabled, and I can choose to manually keep it enabled or not do anything and just let it turn off. I feel like this is a good enough solution to this from GitHub themselves, and that integrating a keepalive workflow for everyone by default would be overkill.

We may document this behaviour and workaround, though, if it's deemed an issue.

**Comment by gmpinder** [comment]
I agree, I think this might be better as a documented issue, and we can point to this GitHub action for it.
